integrations/pagerduty/.port/resources/actions.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "identifier": "trigger_incident",
+    "title": "Trigger incident",
+    "icon": "pagerduty",
+    "description": "Trigger a page in PagerDuty that will alert the on-call to an issue",
+    "trigger": {
+      "type": "self-service",
+      "operation": "DAY-2",
+      "userInputs": {
+        "properties": {
+          "title": {
+            "type": "string",
+            "title": "Title",
+            "description": "Title of the incident"
+          },
+          "description": {
+            "type": "string",
+            "title": "Description",
+            "description": "Description of the incident"
+          }
+        },
+        "required": [],
+        "order": [
+          "title"
+        ]
+      },
+      "blueprintIdentifier": "pagerdutyService"
```

Comment:
Why are we using service instead of incident?

Reply:
A new incident needs to be created with regards to a service whose Id is on the service entity. We are not modifying an existing incident; the action needs to be performed on the service blueprint for us to create an incident about it